### PR TITLE
Added profile fixes discussed in the last meeting

### PR DESCRIPTION
**Pull request by airenzp**
## Description

Solves issues:
Name the top label HOME
Use darker text background and blue background for the tabs
If there is only one choice don't show a menu when selecting radars
Show tooltip on radar table column _difference_ instead of table footnote
Highlight cohorts selection in the home page.

See corresponding PR in sjpp

Home page looks like this now:

Screenshot 2024-12-17 at 12 38 11 PM

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/dom/table.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export type Column = {
 	/** left, center, right. If missing it is aligned to the left by default */
 	align?: string
 	/** tooltip describing column content */
-	title?: string
```
Comment (xzhou82) [inline client/dom/table.ts:36]: with this change, can u try to verify if plot code calling renderTable() is not still supplying `.title`?

Comment (airenzp) [reply client/dom/table.ts:36]: This property was added by me for the profile, I dont think anyone else is using it. Is more clear to name it tooltip.

File: shared/types/src/dataset.ts [hunk 2]
```diff
@@ -785,7 +785,6 @@ export type SelectCohortEntry = {
 	//Therefore when switching cohorts, it is necessary to delete the plots opened and the global filter
 	clearOnChange?: { [index: string]: boolean }
 	//Allows to customize the color of the selected column on the cohort features table. Used by the PrOFILE
```
Comment (xzhou82) [inline shared/types/src/dataset.ts:787]: should move comment along with property

File: client/mass/about.ts [hunk 3]
```diff
@@ -91,14 +91,14 @@ export class MassAbout {
...
 		}
-
 		if (opts.selectCohort?.prompt) {
 			this.dom.cohortPrompt = this.subheader
 				.append('div')
 				.style('margin-left', '10px')
 				.style('padding-top', '30px')
 				.style('padding-bottom', '10px')
 				.style('font-weight', 'bold')
+				.style('font-size', '1.2em')
```
Comment (xzhou82) [inline client/mass/about.ts:101]: this causes radio buttons look noticeably small compared to text
try .style.zoom="120%" on the div element wrapping these

Comment (airenzp) [reply client/mass/about.ts:101]: When I scale, the way to zoom is to use the scale property, it looked like this, I wanted to keep the radio buttons aligned with the text. I will see how.
Screenshot 2024-12-17 at 1 03 54 PM

File: client/mass/about.ts [hunk 4]
```diff
@@ -257,7 +258,7 @@ export class MassAbout {
 			}
 		}
 		const activeColumns = this.dom.cohortTable.selectAll(selector)
-		const color = state.termdbConfig.activeColor || 'yellow'
+		const color = state.termdbConfig.massNav.activeColor || 'yellow'
```
Comment (xzhou82) [inline client/mass/about.ts:261]: massNav is optional

**Review by xzhou82 — APPROVED** [review]

**Pull request merged**